Day-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,9 @@
+.Buttons {
+  display:flex;
+}
+
+.NumRow {
+  display: flex;
+  justify-content: space-between;
+}
+
```

Comment:
You'll want to add this code in, the container will force to calculator to take the width of 500px and have the `.myCalculator` to take 100% width of the container. With `margin: 0 auto` set the calculator in the center.
```
.container {
  max-width: 500px;
  height: auto;
  margin: 0 auto;
}

.container .myCalculator {
  width: 100%;
}
```